satellite/console/service.go

```diff
@@ -475,6 +476,17 @@ func (s *Service) Token(ctx context.Context, email, password string) (token stri
 
 	user, err := s.store.Users().GetByEmail(ctx, email)
 	if err != nil {
+		users, err := s.store.Users().GetAllByEmail(ctx, email)
+		if err != nil {
+			return "", errs.New(internalErrMsg)
+		}
+
+		for _, userToCheck := range users {
+			if err = bcrypt.CompareHashAndPassword(userToCheck.PasswordHash, []byte(password)); err == nil {
+				return "", errs.New(verifyEmailErrMsg)
```

Comment:
If we return here, we skip checking all the rest of the users in the loop. I am assuming if any of the items pass the `CompareHashAndPassword` check, you do not want to return the error. That does not happen right now.

Reply:
Also, why are we comparing passwords to determine whether to send `verifyEmailErrMsg`? If `bcrypt.CompareHashAndPassword(userToCheck.PasswordHash, []byte(password))` fails, does that definitely mean the user is not verified? Is there a different/better way to check?